lang/de_DE.po

```diff
@@ -284,10 +284,8 @@ msgstr ""
 msgid "%s made unknown change"
 msgstr "%s machte unbekannte Änderungen"
 
-#
-# File: src/etc/inc/crypt.inc, line: 47
```

Comment:
most of them got lost when I used "Katalog" -> "Aus POT Datei aktualisieren" when I updated the language file now most of them are lost.